src/commands/ListDiscordLog.ts

```diff
@@ -0,0 +1,42 @@
+import * as discord from "discord.js";
+import { BotgartCommand } from "../BotgartCommand";
+import { log } from "../Util";
+import { TextChannel } from "discord.js";
+
+/**
+Testcases:
+
+*/
+export class ListDiscordLog extends BotgartCommand {
+    constructor() {
+        super("listdiscordlog", {
+                aliases: ["listdiscordlog"],
+                quoted: true,
+                args: [
+                    {
+                        id: "channel",
+                        type: (message: discord.Message | undefined, phrase?: string) => message?.guild?.channels.cache.find(channel => channel.name === phrase)?.name
+                                                                  ?? (message?.channel as discord.TextChannel).name
+                    }
+                    
+                ],
+                userPermissions: ["ADMINISTRATOR"]
+            },
+            {
+                cronable: false,
+                availableAsDM: false
+            }
+        );
+    }
+
+    command(message, responsible, guild, args) {
+        const cl = this.getBotgartClient();
+        let textChannel : TextChannel; 
```

Comment:
We should be as restrictive as possible during variable declarations. `let` declares a variable that is only visible in the current scope `var` creates a variable that would be visible in the surrounding scopes as well (avoid as much as possible) and `const` is for when a variable is never supposed to change. In this case and the case of `types` in line 36, `const` would be the more appropriate modifier. This is a very minor issue, but should be heeded for future commits.